core/src/main/java/org/apache/carbondata/core/datastore/page/encoding/ColumnPageDecoder.java

```diff
@@ -29,6 +31,12 @@
    */
   ColumnPage decode(byte[] input, int offset, int length) throws MemoryException, IOException;
 
+  /**
+   *  Apply decoding algorithm on input byte array and fill the vector here.
+   */
+  ColumnPage decodeAndFillVector(byte[] input, int offset, int length, ColumnVectorInfo vectorInfo,
```

Comment:
decodeAndFillVector return type can be void as ColumnPage returned from decodeAndFillVector will not be used

Reply:
ok